.zuul.yaml

```diff
@@ -0,0 +1,71 @@
+---
+- job:
+    name: openstack-e2e-abstract
+    abstract: true
+    parent: openstack-access-base
+    description: |
+      An abstract job for e2e testing of cluster stacks project.
+      This job is not intended to be run directly, but instead
+      must be inherited from it.
+    pre-run: playbooks/dependencies.yaml
+    run: playbooks/openstack/e2e.yaml
```

Comment:
IMO we should also add some cleanup-run, which is executed when the job is cancelled

Reply:
Agree

Reply:
Sure, I will add that.

Reply:
see 6409a5361bea0654dc743b970f0eb3cb3884375c